documents/policies/external/Competition Requirements.md

```diff
@@ -168,8 +176,7 @@ The WCA Delegate must submit the following information when requesting approval
       1. If there is a deadline, it must not be earlier than the time registration is set to close.
       2. The deadline for adding registered events may be left blank, in which competitors may add registered events up until an event takes place.
       3. Competitors may choose not to compete in an event they are registered for.
-      4. Exception: for competitions with only one event, the deadline for adding registered events may be omitted.
-      5. At the discretion of the WCA Delegate(s), a competitor who is deemed to have made a mistake with the user interface during the registration process (e.g. registering for the inverse list of events that they intended) can be granted an exception to the deadline for adding registered events.
+      4. At the discretion of the WCA Delegate(s), a competitor who is deemed to have made a mistake with the user interface during the registration process (e.g. registering for the inverse list of events that they intended) can be granted an exception to the deadline for adding registered events.
```

Comment:
For the Regs, we generally do not renumber points like this when removing regs or articles. I'm not sure if you have a different history with WCRP, but it is something to consider, since past references can become a pain

Reply:
I think the document compiler wouldn't let you skip numbers in a markdown list.